applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -438,7 +441,14 @@ def _create_builder_and_solver(self):
         linear_solver = self.get_linear_solver()
         if self.settings["block_builder"].GetBool():
             bs_params = self.settings["builder_and_solver_settings"]
-            builder_and_solver = KratosMultiphysics.ResidualBasedBlockBuilderAndSolver(linear_solver, bs_params)
+            consider_lagrange_multiplier_constraint_resolution = self.settings["builder_and_solver_settings"]["consider_lagrange_multiplier_constraint_resolution"].GetString()
+            if consider_lagrange_multiplier_constraint_resolution == "none":
```

Comment:
i really think it is a bad strategy to remove settings... 

in any case the validation is not using them (that's exactly why we shouldn't do recursive validation). They should be validated by the builder_and_solver itself, at least as soon as factories will be merged

Reply:
I know, but the factories are not yet merged...